models/partner/published_data/ptr_dim_person_demographics.sql

```
@@ -0,0 +1,38 @@
+
+Data Quality Filters:
+
+• Excludes persons without birth dates (required for age)
+
+• Excludes persons without any registration history
+
+For historical analysis, use dim_person_demographics_historical.
+For monthly snapshots, use person_month_analysis_base.
+*/
+
+SELECT * FROM {{ref('dim_person_demographics')}}
```

SQLFluff (4.2.2)

[error] Files must end with a single trailing newline.

(LT12)